### PR TITLE
first commit of dashboard task 28

### PR DESCRIPTION
**Pull request by Folarinde**

### CODE REVIEW

File: Dashboard_task_28/style.css [hunk 1]
```diff
@@ -0,0 +1,231 @@
...
+    .date__time{
+        display: flex;
+        padding: .25em;
+        margin: 0 .4em;
+    }
+    .nav__links{
+        margin: 0 1em;
+    }
+}
+@media screen and (max-width: 730px) {
```
Comment (Edema-wxd) [inline Dashboard_task_28/style.css:216]: please create the mobile view for both pages with max width of 380px

**Review by Edema-wxd — COMMENTED** [review]
well done

File: Dashboard_task_28/style.css (hunk 1 above)
Comment (Folarinde) [reply Dashboard_task_28/style.css:216]: Okay! Right away.

**Comment by Edema-wxd** [comment]
your page rendered nicely but there are a few things to take note of

Dashboard main & others

- the texts in the buttons on the side nav bar, make them to be center aligned with their respective image/icon explore the css float property
- increase the spacing between the about us text and the + sign, apply to help center also
- insert icon for menu on the popular categories section
- reduce the spacing on the nav bar, the max , and the page shouldn't scroll
- add shadow and color of the boxes( popular category ) to match design on figma [board](https://www.figma.com/file/DaCupcCqB7ZJ3ixvHBsTyi/Proj-QR-Main-board?node-id=29%3A3)  

desktop height  = 1024 px
desktop width  = 1440 px
mobile height = 812 px
mobile width = 375 px

Thank you,
please also start work on the mobile view

**Pull request merged**